react/proptypes/shapes.js

```diff
@@ -16,3 +16,22 @@ export const paymentShape = PropTypes.shape({
   value: PropTypes.number.isRequired,
   installments: PropTypes.number.isRequired,
 })
+
+// This should be made available by `@vtex/delivery-packages`
```

Comment:
Remove comment and create issue at the repo.

Reply:
Issue created and comment removed